go.mod

```diff
@@ -76,7 +77,7 @@ require (
 	github.com/go-openapi/jsonreference v0.19.6 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
-	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
➜  weaviate git:(evaluating-grpc) ✗ go get github.com/golang/protobuf
go: module github.com/golang/protobuf is deprecated: Use the "google.golang.org/protobuf" module instead.

Reply:
Ignoring this for now, as it's hard to see where that indirect dependency comes from. There also seem to be various protobuf versions used throughout Weaviate. This should be cleaned up at some point.